### ISSUE TITLE
Unable to read files and low resolution.

### ISSUE THREAD

**Issue by Caffin8tor** [CypherpunkArmory/GIMP#13]
I've used Gimp several times on my phone by using an external monitor with Samsung DeX and it was working fine until a day ago. Now, when trying to browse for files, only directories appear in the file dialog. It appears the app lacks permissions to even view file names. If I try to enter a filename for a file I know is present, there is a permission denied error. Another issue which is not as serious, but started occuring at the same time is that the app will only start up in 1000x590 resolution despite the desktop resolution being larger. I've tried restarting my device and reinstalling Gimp but that did not work.

**Comment by Caffin8tor** [comment]
It looks like a recent update resolved the low resolution issue, but the app is currently useless as it is still unable to access any files on the device. It appears that the app needs permissions from the OS security for file access, but never requests access. There is no option to add Gimp to the apps allowed to access files in the OS permissions settings either. This may have even been a security loophole that was recently fixed by Google because I don't think the app ever asked for file access before either. Another tip about the issue is that files do not appear in the shell console even when using "sudo" to run commands with root permissions. Directories appear to show up just fine, but all files are hidden from the app.

**Comment by sukhmel** [comment]
Yeah, until this is fixed, the app is useless, stumbled upon this today, too
